### ISSUE TITLE
node: install takes a long time, builds from scratch

### ISSUE THREAD

**Issue by zenazn** [duckdb/duckdb#9113]
### What happens?

Installing duckdb's nodejs bindings takes a very long time (on my computer, an M1 MBA, over 11 minutes), because npm re-compiles DuckDB at install time.

I would expect duckdb to use a pre-compiled binary, which would result in ~instant installs. I believe `node-pre-gyp` has a default binary bucket of some sort, but I'd also be willing to pass an explicit environment variable (https://github.com/mapbox/node-pre-gyp#download-binary-files-from-a-mirror) if duckdb wants to host their mirror elsewhere.

### To Reproduce

In a new directory:
```
npm init -y
time npm install --save duckdb
```

You can easily verify that it's e.g., `clang` taking up the CPU time

### OS:

M1 Mac

### DuckDB Version:

0.9.0

### DuckDB Client:

nodejs

### Full Name:

Carl Jackson

### Affiliation:

Watershed

### Have you tried this on the latest `main` branch?

I have tested with a release build (and could not test with a main build)

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] Yes, I have

**Comment by carlopi** [comment]
Node CI job for the release has failed without deploying the right artefacts, and currently it's in a state that requires manual builds.

We are aware of this, it's lagging behind the other builds, but might need another day or two to sort this out properly on all platforms.

**Comment by archiewood** [comment]
Came here to post this issue earlier. Got distracted.
Came back 5h later. Someone had already done it. 
❤️ duckdb

**Comment by MikeNitsenko** [comment]
`"duckdb": "^0.9.1-dev19.0"` seems to work fine, waiting for `0.9.1`? :wink:

**Comment by hannes** [comment]
We hope to have the binaries up next week. Cheers

**Comment by RandomFractals** [comment]
@hannes any update on this yet? also, we see you moved duckdb node to new repo. should we be checking that repo for new release updates instead?

https://github.com/duckdb/duckdb-node

**Comment by Mause** [comment]
v0.9.1 has been released to npm now, the builds are going up: https://github.com/duckdb/duckdb-node/actions/runs/6533082665

Please feel free to raise a new issue if you have any issues with it

**Comment by amir20** [comment]
I am seeing this behavior again. I am on M3 Macbook air. I do `npm i duckdb` and notice that it takes a very long time. Look at the process list, seems like duckdb is built. 

Did something change?

**Comment by amir20** [comment]
Seems like `@duckdb/node-api` works. Not sure if `duckdb` is deprecated.

**Comment by hannes** [comment]
Yup can confirm this currently builds from source, will investigate

**Comment by hannes** [comment]
In general though, the long-term plan is to deprecate the current package, yes.

**Comment by hannes** [comment]
see https://github.com/duckdb/duckdb-node/pull/148
